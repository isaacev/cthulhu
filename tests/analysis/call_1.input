use IO;

fn main() -> Void {
  IO::println();
}
