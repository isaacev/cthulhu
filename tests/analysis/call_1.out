
  CALL WITH THE WRONG NUMBER OF ARGUMENTS  -----------------------------------

  Expected 1 total arguments, found 0 instead:

    : call_1
  4 |   Io::println();
    |   ^^^^^^^^^^^^^

